apps/condo/domains/miniapp/schema/SendB2BAppPushMessageService.js

```diff
@@ -0,0 +1,218 @@
+                const {
+                    data: {
+                        dv,
+                        sender,
+                        user: userFilter,
+                        organization: organizationFilter,
+                        app: b2bAppFilter,
+                        type,
+                        meta,
+                    },
+                } = args
+                const authedItemId = get(context, 'authedItem.id', null)
+
+                const [appSettings] = await itemsQuery('AppMessageSetting', {
+                    where: {
+                        b2bApp: { ...b2bAppFilter, deletedAt: null },
+                        type,
+                        deletedAt: null,
+                    },
+                    first: 1,
+                })
+                if (appSettings && appSettings.isBlacklisted) {
+                    throw new GQLError(ERRORS.APP_IN_BLACK_LIST, context)
+                }
+
+                await redisGuard.checkCustomLimitCounters(
+                    `sendB2BAppPushMessage:${type}:app:${b2bAppFilter.id}:org:${organizationFilter.id}:user:${userFilter.id}`,
+                    get(appSettings, 'notificationWindowSize') || DEFAULT_NOTIFICATION_WINDOW_DURATION_IN_SECONDS,
+                    get(appSettings, 'numberOfNotificationInWindow') || DEFAULT_NOTIFICATION_WINDOW_MAX_COUNT,
+                    context,
+                )
+
+                const [b2bAppContext] = await itemsQuery('B2BAppContext', {
+                    where: {
+                        organization: { ...organizationFilter, deletedAt: null },
+                        app: { ...b2bAppFilter, deletedAt: null },
+                        status: CONTEXT_FINISHED_STATUS,
+                        deletedAt: null,
+                    },
+                    first: 1,
+                })
+                if (!b2bAppContext) {
+                    throw new GQLError(ERRORS.NO_B2B_CONTEXT, context)
+                }
+
+                const [b2bAppAccessRight] = await itemsQuery('B2BAppAccessRight', {
+                    where: {
+                        accessRightSet: {
+                            app: { ...b2bAppFilter, deletedAt: null },
+                            canExecuteSendB2BAppPushMessage: true,
+                            deletedAt: null,
+                        },
+                        user: { id: authedItemId, deletedAt: null },
+                        deletedAt: null,
+                    },
+                    first: 1,
+                })
+                if (!b2bAppAccessRight) {
+                    throw new GQLError(ERRORS.NO_B2B_APP_ACCESS_RIGHT, context)
+                }
+
+                const [organization] = await itemsQuery('Organization', {
+                    where: {
+                        ...organizationFilter,
+                        deletedAt: null,
+                    },
+                    first: 1,
+                })
```

Comment:
this request will be removed after i change locale getting from user instead organization